lib/ibanity/api/sandbox/financial_institution_transaction.ex

```diff
@@ -169,6 +169,65 @@ defmodule Ibanity.Sandbox.FinancialInstitutionTransaction do
     |> Client.execute(:get, @api_schema_path)
   end
 
+  @doc """
+  [Update transaction](https://documentation.ibanity.com/xs2a/api#update-financial-institution-transaction)
+
+  See `update/1`
+
+  ## Example
+      iex> attributes = [
+      ...>   value_date: "2017-05-22T00:00:00Z",
+      ...>   execution_date: "2017-05-25T00:00:00Z",
+      ...>   amount: 84.42,
+      ...>   remittance_information_type: "NEW SHOES",
+      ...>   remittance_information: "unstructured",
+      ...>   ...
+      ...> ]
+      ...> FinancialInstitutionTransaction.update(
+      ...>   "ad6fa583-2616-4a11-8b8d-eb98c53e2905",
+      ...>   "740b6ae8-a631-4a32-9afc-a5548ab99d7e",
+      ...>   "d9d60751-b741-4fa6-8524-8f9a066ca037",
+      ...>   "b59c4b50-3188-41df-9f69-92777a056fe6",
+      ...>   attributes
+      ...> )
+      {:ok, %Ibanity.FinancialInstitutionTransaction{id: "44cd2dc8-163a-4dbe-b544-869e5f84ea54", ...}}
+  """
+
+  def update(
+        %Request{} = request,
+        financial_institution_id,
+        financial_institution_user_id,
+        financial_institution_account_id,
+        financial_institution_transaction_id,
+        attributes
+      ) do
+    request
+    |> Request.attributes(attributes)
+    |> Request.id(:financial_institution_id, financial_institution_id)
+    |> Request.id(:financial_institution_user_id, financial_institution_user_id)
+    |> Request.id(:financial_institution_account_id, financial_institution_account_id)
+    |> Request.id(:id, financial_institution_transaction_id)
+    |> Client.execute(:patch, @api_schema_path)
```

Comment:
You could use `|> update()` here  instead of `|> Client.execute...` which would also add the resource type to the request.